README.md

```diff
@@ -24,6 +24,9 @@
    $ pip install --user azure-cli-shell
 ```
 
+A list of common install issues and their resolutions are available at [install troubleshooting](https://github.com/Azure/azure-cli-shell/blob/master/doc/install_troubleshooting.md).
```

Comment:
Think this url would be wrong.
The file below looks like the directory is docs.